comiccrawler/mods/cartoonmad.py

```diff
@@ -22,7 +22,7 @@ def get_episodes(html, url):
 	return s
 
 def get_images(html, url):
-	return re.search('img src="(http://[\w.]*?cartoonmad\.com.+?)"', html).group(1)
+	return "https://www.cartoonmad.com/comic/comicpic.asp" + re.search('img src="comicpic.asp(.+?)"', html).group(1)
```

Comment:
Instead of concatenating the string, use `urljoin` to build the URL:
```py
pic_url = re.search('img src="(comicpic[^"]+)', html).group(1)
return urljoin(url, pic_url)
```

Reply:
OK